nacossync-worker/src/main/java/com/alibaba/nacossync/pojo/model/TaskDO.java

```diff
@@ -28,7 +28,7 @@
  */
 @Data
 @Entity
-@Table(name = "task")
+@Table(name = "task", uniqueConstraints = {@UniqueConstraint(columnNames={"taskId"})})
```

Comment:
No need to add uniqueConstraints for taskID since it is already under uniqueConstraints

Reply:
Ok，   "uniqueConstraints" is used to add unique index in the mysql ,  but there is no work in hibernate because of some bug.   So i will add unique index manually, and this annotation not need any more